File: SE4-P/ejer4.py
import math

print("--- SISTEMA DE CALCULO DE ÁREAS ---\n")

print("1. Área del cuadrado")
print("2. Área del triángulo")
print("3. Área del rectángulo")
print("4. Área del círculo\n")

opc = int(input("Ingrese una opción: "))

match opc:
    case 1:
        l = int(input("\nIngresa el lado del cuadrado: "))
        area = l * l
        print("\nÁrea del cuadrado: ",area)
    case 2:
        b = int(input("\nIngresa la base del triángulo: "))
        h = int(input("Ingresa la altura del triángulo: "))
        area = b * h / 2
        print("\nÁrea del triángulo: ", area)
    case 3:
        br = int(input("\nIngresa la base del rectángulo: "))
        hr = int(input("Ingresa la altura del rectángulo: "))
        area = br * hr
        print("\nÁrea del triángulo: ", area)
    case 4:
        r = int(input("\nIngresa la radio del circulo: "))
        area = math.pi * math.pow(r, 2)
        print("\nÁrea del circulo: ", area)
    case _: print("\nOpción incorrecta!")

File: SEM4-C/ejer4.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SEM4_C
{
    internal class ejer4
    {
        static void Main(string[] args)
        {
            double area;
            Console.WriteLine("--- SISTEMA DE CALCULO DE ÁREAS ---\n");

            Console.WriteLine("1. Área del cuadrado");
            Console.WriteLine("2. Área del triángulo");
            Console.WriteLine("3. Área del rectángulo");
            Console.WriteLine("4. Área del círculo\n");

            Console.Write("Ingrese una opción: ");
            int opc = int.Parse(Console.ReadLine());

            switch (opc)
            {
                case 1:
                    Console.Write("\nIngresa el lado del cuadrado: ");
                    int l = int.Parse(Console.ReadLine());
                    area = l * l;
                    Console.WriteLine("\nÁrea del cuadrado: "+area);
                    break;
                case 2:
                    Console.Write("\nIngresa la base del triángulo: ");
                    int b = int.Parse(Console.ReadLine());
                    Console.Write("Ingresa la altura del triángulo: ");
                    int h = int.Parse(Console.ReadLine());
                    area = b * h / 2;
                    Console.WriteLine("\nÁrea del triángulo: " + area);
                    break;
                case 3:
                    Console.Write("\nIngresa la base del rectangulo: ");
                    int br = int.Parse(Console.ReadLine());
                    Console.Write("Ingresa la altura del rectangulo: ");
                    int hr = int.Parse(Console.ReadLine());
                    area = br * hr;
                    Console.WriteLine("\nÁrea del triángulo: " + area);
                    break;
                case 4:
                    Console.Write("\nIngresa la radio del circulo: ");
                    int r = int.Parse(Console.ReadLine());
                    area = Math.PI * Math.Pow(r, 2);
                    Console.WriteLine("\nÁrea del circulo: " + area);
                    break;
                default:
                    Console.WriteLine("\nOpción incorrecta!");
                    break;
            }
        }
    }
}
